### PR TITLE
User/markackert/chat docs additions

### PR DESCRIPTION
**Pull request by MarkAckert**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/zowe-chat/chat_configure_server.md [hunk 1]
```diff
@@ -74,3 +83,62 @@ You can configure the Zowe Chat server by editing the `chatServer.yaml` configur
...
+   # Specify the port number of your z/OSMF server.
+   # The default value is 443.
+   port: 443
+
+   # Specify whether self-signed certificates are rejected or not. The value can be true or false.
+   # The default value is true
+   rejectUnauthorized: true
+
+   # Specify authentication type. The value can be token or password
+   # The default value is password
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/chat_configure_server.md:116]: need update

File: docs/user-guide/zowe-chat/systemrequirements-chat.md [hunk 2]
```diff
@@ -1,44 +1,83 @@
...
-   - Microsoft Teams. See [Configuring Microsoft Teams chat platform](chat_prerequisite_teams.md).
-  
-   - Slack. See [Configuring Slack chat platform](chat_prerequisite_slack.md).
-       
+
+### Node.js
+
+- Node.js v16.x. Zowe Chat has not yet been tested with 14.x or 18.x.
+
+  Node may not be included out of the box in your Linux distribution; if so you must install it.  To install Node.js follow the instructions on the [Node.js Download Page](https://nodejs.org/en/download/). It is recommended to use a package manager if possible, [as outlined here](https://nodejs.org/en/download/package-manager/).
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/systemrequirements-chat.md:28]: need updates

File: docs/user-guide/zowe-chat/systemrequirements-chat.md [hunk 3]
```diff
@@ -1,44 +1,83 @@
...
+## z/OS System Requirements
+### z/OSMF
+
+- IBM z/OS Management Facility (z/OSMF) Version 2.2, Version 2.3 or Version 2.4.
+
+  - z/OSMF is included with z/OS so does not need to be separately installed. It does need to be configured with REST APIs enabled, as these APIs are used by the Zowe Chat application. 
+
+  **Tips:**
+
+  - For non-production use of Zowe (such as development, proof-of-concept, demo), you can customize the configuration of z/OSMF to create what is known as "z/OS MF Lite" that simplifies the setup of z/OSMF. As z/OS MF Lite only supports selected REST services (JES, DataSet/File, TSO and Workflow), you will observe considerable improvements in startup time as well as a reduction in the efforts involved in setting up z/OSMF. For information about how to set up z/OSMF Lite, see [Configuring z/OSMF Lite (non-production environment)](../systemrequirements-zosmf-lite.md).
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/systemrequirements-chat.md:39]: 1. non-production use of Zowe Chat
2. Wording of z/OS MF Lite

**Review by nannanli — APPROVED** [review]
thank you @MarkAckert for the updates! I made some edits and will merge them in.

**Pull request merged**